File: 구은성/9월/0923/77884.java
class Solution {
    public int solution(int left, int right) {
        int answer = 0;
        
        for(int i = left; i <= right; i++) {
            if(couDiv(i) % 2 == 0) {
                answer += i;
            }
            else {
                answer -= i;
            }
        }
        
        return answer;
    }
    
    public int couDiv(int num) {
        int cou = 0;
        for (int i = 1; i <= num; i++) {
            if (num % i == 0) {
                cou++;
            }
        }
        return cou;
    }
}

File: 김강민/10월/Week_02/1013/77884.py
def solution(left, right):
    answer = 0
    
    for i in range(left, right + 1):
        cnt = 0
        for j in range(1, i + 1):
            if i % j == 0:
                cnt += 1
                
        if cnt % 2 == 0:	
            answer += i
        else:
            answer -= i
    return answer